src/libstd/primitive_docs.rs

```diff
@@ -10,8 +10,46 @@
 
 #[doc(primitive = "bool")]
 //
-/// The boolean type.
+/// The boolean type represents a value, which could only be either `true` or `false`.
```

Comment:
I would like to move this addition into the body, rather than being the main comment. Symmetry: http://doc.rust-lang.org/std/#primitives